plugins/dbus_service/dbus_service_tag.c

```diff
@@ -528,6 +535,79 @@ dbus_service_tag_complete_pending_calls(
     }
 }
 
+static
+gboolean
+dbus_service_tag_acquire(
+    OrgSailfishosNfcTag* iface,
+    GDBusMethodInvocation* call,
+    gboolean wait,
+    gboolean force_presence_check,
+    DBusServiceTagPriv* self,
+    DBusAcquireTagCompleteFunc complete)
+{
+    DBusServiceTag* pub = &self->pub;
+    NfcTarget* target = pub->tag->target;
+    DBusServiceTagLock* current_lock = self->lock;
+    const char* name = g_dbus_method_invocation_get_sender(call);
+
+    if (current_lock && !g_strcmp0(current_lock->name, name)) {
+        /* This client already has the lock */
+        current_lock->count++;
+        GDEBUG("Lock request from %s (%u)", name, current_lock->count);
+        nfc_target_sequence_set_flags(current_lock->seq,force_presence_check ?
```

Comment:
```
dbus_service/dbus_service_tag.c: In function ‘dbus_service_tag_acquire’:
dbus_service/dbus_service_tag.c:557:9: warning: implicit declaration of function ‘nfc_target_sequence_set_flags’; did you mean ‘nfc_target_sequence_get_flags’? [-Wimplicit-function-declaration]
  557 |         nfc_target_sequence_set_flags(current_lock->seq,force_presence_check ?
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |         nfc_target_sequence_get_flags
```
`nfc_target_sequence_set_flags()` has to be made public since it's required by a plugin.

Reply:
And a space after comma please 🙂